sim/core/racials.go

```diff
@@ -92,7 +92,8 @@ func applyRaceEffects(agent Agent) {
 		character.PseudoStats.ReducedFrostHitTakenChance += 0.02
 
 		// Gun specialization (+1% ranged crit when using a gun).
-		if character.Ranged().RangedWeaponType == proto.RangedWeaponType_RangedWeaponTypeGun {
+		ranged := character.Ranged()
+		if ranged != nil && ranged.RangedWeaponType == proto.RangedWeaponType_RangedWeaponTypeGun {
 			character.AddBonusRangedCritPercent(1)
 		}
```

Comment:
Just a note that we'll need to update these racials for MoP, Crack Shot now provides 1% Expertise when a ranged weapon is equipped.